deploy/releases/0.8.0/readme.adoc

```diff
@@ -0,0 +1,38 @@
+= Runtime Component Operator v0.8.0
+
+== Changelog
+
+All notable changes are documented in the link:++https://github.com/application-stacks/runtime-component-operator/blob/main/CHANGELOG.md#080++[Changelog].
+
+== Supported Environments
+
+The Runtime Component Operator is tested and built on the following CPU architectures:
+
+* amd64
+
+The operator will run on any cluster that has Kubernetes v1.19 or higher.
+
+== Installation
```

Comment:
Should this section explicitly state 'Installation on OCP' or something similar?